web/pages/api/contact.js

```diff
@@ -24,11 +24,13 @@ export default async function handler(req, res) {
         `<p>Name: ${req.body.name}</p>
         <p>Email: ${req.body.emailAddress}</p>
         <p>Phone: ${req.body.phoneNumber}</p>
-        <p>Message: ${req.body.message}</p>`,
+        <p>Message: ${req.body.message}</p>
+        <p>Preference: ${req.body.preference}</p>`,
         `Name: ${req.body.name}\n
         Email: ${req.body.emailAddress}\n
         Phone: ${req.body.phoneNumber}\n
-        Message: ${req.body.message}`
+        Message: ${req.body.message}\n
+        Preference: ${req.body.preference}`
```

Comment:
Same here as above

Reply:
I added a `.join(', ') ` to each of these.